resources/css/style.css

```diff
@@ -62,3 +62,7 @@
       width: 90% !important;
   }
 }
+
+div.card > div.card-action > span.chip {
```

Comment:
This is really really verbose and still might not target only the clickable chips ...

Try `.clickable .chip`

You need to [look at the markup](https://github.com/coala/projects/blob/7ea691b5a03eb9c4d4a3c23038b0ce99a18fcf27/partials/tabs/projects.html#L25) thoroughly ;)